string user_id
time time
string date
string task_name
string demo_id

dviz_core/Step[] steps